tests/FSharp.Core.UnitTests/FSharp.Core.UnitTests.fsproj

```diff
@@ -25,11 +25,6 @@
     <TailCalls>true</TailCalls>
   </PropertyGroup>
 
-  <PropertyGroup>
```

Comment:
Not used, not needed.

This was introduced in [this PR](https://github.com/dotnet/fsharp/pull/6151/files#diff-f98e0fa7b5084c43c3d716473bb6955e0aa574fd9aae56d97329b0a815842d9d) 5 years ago but then was stopped being used in [this change](https://github.com/dotnet/fsharp/pull/9992/files#diff-61f90544c18b10b8bedcd6538a8786d2e44f29da09da472871a33d5964832591) a year after.